src/account/AccountLoupe.sol

```diff
@@ -52,14 +52,14 @@ abstract contract AccountLoupe is IAccountLoupe {
     }
 
     /// @inheritdoc IAccountLoupe
-    function getPreValidationHooks(bytes4 selector)
+    function getPreValidationHooks(FunctionReference validationFunction)
```

Comment:
I think I reviewed this part before in one of @adamegyed 's PRs. Is this PR ready to be reviewed?